pkg/sass-parser/lib/src/dynamic-import.test.ts

```diff
@@ -0,0 +1,155 @@
+// Copyright 2025 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import {DynamicImport, ImportList, ImportRule, sass, scss} from '..';
+
+describe('a dynamic import', () => {
+  let node: DynamicImport;
+  beforeEach(() => void (node = new DynamicImport({url: 'foo'})));
```

Comment:
Is this beforeEach() necessary? I see that `node` gets instantiated by `describeNode`'s `create()` callback, and then I don't see other uses of `node`